apps/admin/src/components/main/FormList/FormList.tsx

```diff
@@ -0,0 +1,118 @@
+import ListHeader from '@/components/common/TableHeader/TableHeader';
+import ListItem from '@/components/common/TableItem/TableItem';
+import { Column, Row, Text } from '@maru/ui';
+import { styled } from 'styled-components';
+
+import { FormType, GraduationType, Status } from '@/types/notice/client';
+
+const FORM_DATA: {
+    id: number;
+    name: string;
+    birthday: string;
+    graduationType: GraduationType;
+    school: string;
+    status: Status;
+    type: FormType;
+}[] = [
+    {
+        id: 0,
+        name: '김밤돌',
+        birthday: '2005-04-15',
+        graduationType: 'EXPECTED',
+        school: '비전중학교',
+        status: '최종 제출됨',
+        type: 'REGULAR',
+    },
+    {
+        id: 1,
+        name: '김밤돌',
+        birthday: '2005-04-15',
+        graduationType: 'QUALIFICATION_EXAMINATION',
+        school: '비전중학교',
+        status: '반려됨',
+        type: 'REGULAR',
+    },
+    {
+        id: 2,
+        name: '김밤돌',
+        birthday: '2005-04-15',
+        graduationType: 'EXPECTED',
+        school: '비전중학교',
+        status: '최종 제출됨',
+        type: 'MULTICULTURAL',
+    },
+];
+
+const KoreanFormType: { [key in FormType]: string } = {
+    REGULAR: '일반',
+    MEISTER_TALENT: '마이스터인재',
+    NATIONAL_BASIC_LIVING: '국민기초생활수급자',
+    NEAR_POVERTY: '차상위계층',
+    NATIONAL_VETERANS: '국가보훈대상자 (국가유공자)',
+    ONE_PARENT: '한부모가정',
+    FROM_NORTH_KOREA: '북한이탈주민',
+    MULTICULTURAL: '다문화가정',
+    TEEN_HOUSEHOLDER: '소년소녀가장',
+    MULTI_CHILDREN: '다자녀가정자녀',
+    FARMING_AND_FISHING: '농어촌지역출신자',
+    SPECIAL_ADMISSION: '특례입학대상자전형',
+};
```

Comment:
```suggestion
const KoreanFormType: Record<FormType, string> = {
    REGULAR: '일반',
    MEISTER_TALENT: '마이스터인재',
    NATIONAL_BASIC_LIVING: '국민기초생활수급자',
    NEAR_POVERTY: '차상위계층',
    NATIONAL_VETERANS: '국가보훈대상자 (국가유공자)',
    ONE_PARENT: '한부모가정',
    FROM_NORTH_KOREA: '북한이탈주민',
    MULTICULTURAL: '다문화가정',
    TEEN_HOUSEHOLDER: '소년소녀가장',
    MULTI_CHILDREN: '다자녀가정자녀',
    FARMING_AND_FISHING: '농어촌지역출신자',
    SPECIAL_ADMISSION: '특례입학대상자전형',
};
```
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

Record를 사용해보는건 어떨까요?